### ISSUE TITLE
ObservableHQ embedded pages does not work in Samsung TV browser anymore

### ISSUE THREAD

**Issue by andreassolberg** [observablehq/feedback#536]
**Describe the bug**

I have been using observablehq a lot to display visualizations at TV displays at home and at work.
I used export › embed › with a single "chart" cell. I typically use dakboard to create a tile board of multiple visualizations, it displays observablehq charts using iframes.

Lately it have stopped working. I don't know if it is due to changes in Samsung TV OS or at observablehq.

**To Reproduce**

Open a web page on a Samsung TV (I have tested on two different Samsung Frame) containing observablehq charts.

**Expected behavior**
Content should display as in a regular browser.

What happens is that only footer is shown, and not content of the cells.

**Desktop (please complete the following information):**

Samsung TV OS. Latest version.

**Comment by mootari** [comment]
Could you please try to find out the browser and version that your TV is running? According to http://kangax.github.io/compat-table/es6/ the Samsung Browser does support all language features.

Also, does this happen with every notebook?

**Comment by andreassolberg** [comment]
I accessed 
https://observablehq.com/@categorise/circle-legend
at 20:00 CET,  Europe/Amsterdam

The problem occur on all notebooks. And I have tested on two different Samsung Frame TVs.

**Comment by andreassolberg** [comment]
![IMG_0109](https://user-images.githubusercontent.com/101361/217353880-816287c1-b2d3-4425-8235-996bd9351549.jpg)

Here is more browser information.

**Comment by andreassolberg** [comment]
> Also, does this happen with every notebook?

Yes all notebook. But a while ago observablehq did work. I did not recognize exactly when it stopped working. I though it was some of the javascript libraries I used in the notebook that was not compatible, but when I did some research I see that it is related to observablehq it self and that not any notebook does work.

**Comment by CobusT** [comment]
I know this sounds like an unlikely fix, but could you try the suggestion in this thread?

https://us.community.samsung.com/t5/4K-8K-and-Other-TVs/Internet-Browser-on-my-Samsung-TV-not-working/td-p/1382104
